migrations/20201015005801-update-user-roles.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'users',
+          'roleId',
+          { type: Sequelize.DataTypes.INTEGER },
+          { transaction: t }
+        ),
+        queryInterface.sequelize.query(
```

Comment:
I'm curious: is there no Sequelize functionality for this type of update? Is a raw SQL query absolutely necessary?

Reply:
hmm i'm not too familiar with all the functions but I copied this format from another previous migration that I think peter did haha

https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html from this list, it doesn't look like any of the functions can really achieve the same thing (other than maybe `bulkUpdate`? but I don't think that's what it does)